app/controllers/employees_controller.rb

```diff
@@ -2,20 +2,32 @@ class EmployeesController < ApplicationController
   before_action :find_employee, only: [:show, :edit, :update, :destroy]
 
   def index
-    @employee = Employee.all
+    @employees = Employee.all
   end
 
   def show
+    @vacation = Vacation.new
+    @vacations = @employee.vacations
+    @vacation_days = @employee.vacation_days
+    @used_vacation_days = @employee.used_vacation_days
+    @remaining_vacation_days = @employee.remaining_vacation_days
+
+    @position_history = PositionHistory.new
+    @position = @employee.position_histories.last
+    @position_histories = @employee.position_histories
+
+    @all_vacations = @employee.vacations.sum { |vacation| (vacation.finished_on - vacation.started_on).to_i }
```

Comment:
NO really the correct name for the variable. It stores the total number of days spent on vacation. `@days_on_vacation`, `@total_vacation_days`